File: java/menor_de_tres.java
import java.util.Locale;
import java.util.Scanner;

public class menor_de_tres {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int a, b, c, menor;

	    System.out.print("Primeiro valor: ");
	    a = sc.nextInt();
		
	    System.out.print("Segundo valor: ");
	    b = sc.nextInt();
		
	    System.out.print("Terceiro valor: ");
	    c = sc.nextInt();
		

	    if (a < b && a < c) {
	        menor = a;
	    }
	    else if (b < c) {
	        menor = b;
	    }
	    else {
	        menor = c;
	    }
		
	    System.out.printf("MENOR = %d\n", menor);

		sc.close();
	}
}

File: python/menor_de_tres.py
a: int; b: int; c: int; menor: int

a = int(input("Primeiro valor: "))
b = int(input("Segundo valor: "))
c = int(input("Terceiro valor: "))

if a < b and a < c:
	menor = a
elif b < c:
	menor = b
else:
	menor = c

print(f"MENOR = {menor}")